.changeset/new-ties-change.md

```diff
@@ -0,0 +1,5 @@
+---
+'@astrojs/cloudflare': patch
```

Comment:
The PR comment says `major`. I think that's probably appropriate given that this is going to load different code now.

Reply:
first I did do major, but did not know what the implications are. So then I thought it is just a patch.

Reply:
Let's do major just to be safe.